tstyche.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://tstyche.org/schemas/config.json",
   "testFileMatch": [
-    "packages/**/__typetests__/*.test-d.*"
+    "packages/**/__typetests__/*.tst.*"
```

Comment:
The `**/*.tst.*` files would be picked up [default](https://tstyche.org/reference/config-file#test-file-match), but setting this explicitly makes the crawling slightly more efficient.